sdk/test-utils/testing-recorder-new/test/core-v1-test.spec.ts

```diff
@@ -45,7 +46,11 @@ describe("Core V1 tests", () => {
   });
 
   it("storage-queue create queue", async function() {
-    const client = new QueueServiceClient(env.STORAGE_SAS_URL, undefined, { httpClient: recorder });
+    const client = new QueueServiceClient(
+      env.STORAGE_SAS_URL || "undefined",
```

Comment:
would it be better to fail the test if we expect some valid url, instead of passing "undefined" to constructor?

Reply:
Hmmm